boards/arm/bl653_dvk/Kconfig.defconfig

```diff
@@ -17,9 +17,6 @@ config BT_CTLR
 
 if DAC
 
-config DAC_MCP4725
```

Comment:
I think you can remove the entire `if DAC [...] endif` section (here and for some other boards). I2C should also be enabled via devicetree and `DAC_INIT_PRIORITY` is now obsolete for this driver, as it uses its own init priority, see [here](https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/dac/Kconfig.mcp4725).

Reply:
So I2C bus is NOT enabled by devicetree.

Reply:
I added a commit to remove the DAC_INIT_PRIORITY.  However I think the I2C and SPI bus bits is still needed.